src/web/components/Navigation/SideNav.tsx

```diff
@@ -35,9 +35,9 @@ function MenuItem({
 
 export type SideNavProps = Readonly<{
   standardMenu: PortalRoute[];
-  adminMenu: PortalRoute[];
```

Comment:
Decided to rename this to `uid2SupportMenu` everywhere so we didn't get it confused with the 'admin' role since it is only the 'uid2support` role that can view this menu